iceoryx_hoofs/testing/include/iceoryx_hoofs/testing/timing_test.hpp

```diff
@@ -74,7 +76,7 @@
         EXPECT_TRUE(testResult);                                                                                       \
         if (!testResult)                                                                                               \
         {                                                                                                              \
-            std::cout << "\n" << errorMessages << std::endl;                                                           \
```

Comment:
I'm not quite sure but I guess here should also std::cout be used instead of the logger.